lib/CAPI/CMakeLists.txt

```diff
@@ -11,9 +11,37 @@ add_mlir_public_c_api_library(TorchMLIRCAPI
   LINK_COMPONENTS
   Core
 
+  DEPENDS
+  MLIRAffinePassIncGen
```

Comment:
Just so I understand, can you comment on how you came to conclude that these needed to be depended on? Ordinarily, I only see explicit dependencies on the generated files from internal to a target and would have expected that we add the actual transform/pass target itself to the LINK_LIBS PUBLIC.

(you may be right to do this but double checking)

Reply:
(I would have expected the LINK_LIBS depends below to have transitively insured generation of these files)

Reply:
Ok, fine to be explicit like this for now, but can you add a TODO to that effect so we can better diagnose it?